backend/packages/worker/src/worker/parsing/models.py

```
@@ -0,0 +1,14 @@
+from typing import List, Optional
```

Ruff (0.15.11)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)